workflows/common/structs.wdl

```diff
@@ -25,11 +27,21 @@ struct SmrtcellInfo {
 
 struct SampleInfo {
   String name
-  Array[String?] parents
+  Boolean? affected
+  Array[String] parents
```

Comment:
@cbi-star for Singletons will you still need to provide any empty array for the parents?